### PR TITLE
Fix for DuplicateAttributeException: https://wso2.org/jira/browse/CEP-1309

### PR DESCRIPTION
**Pull request by grainier**
Fix for : https://wso2.org/jira/browse/CEP-1309 (DuplicateAttributeException when same attribute name used in correlation, meta and payload data.)

### CODE REVIEW

File: components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/EventManagementUtil.java [hunk 1]
```diff
@@ -37,10 +37,14 @@ public static StreamDefinition constructStreamDefinition(String syncId, org.wso2
 
         List<Attribute> attributes = new ArrayList<Attribute>();
         if (inStreamDefinition.getMetaData() != null) {
-            attributes.addAll(inStreamDefinition.getMetaData());
+            for (Attribute attr : inStreamDefinition.getMetaData()) {
+                attributes.add(new Attribute("meta_" + attr.getName(), attr.getType()));
```
Comment (lasanthafdo) [inline components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/EventManagementUtil.java]: @grainier 
Can you change these to string constants and refer to the constant from here? There already should exist some of these constants in these components.

Comment (grainier) [inline components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/EventManagementUtil.java]: @lasanthafdo updated with constants.

File: components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/util/ConfigurationConstants.java [hunk 2]
```diff
@@ -90,4 +90,7 @@ private ConfigurationConstants() {
     public static enum HAMode {Active, Passive, Backup}
 
     public static final String STORM_EVENT_PUBLISHER_SYNC_MAP = "stormEventPublisherSyncMap";
+
```
Comment (mohanvive) [inline components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/util/ConfigurationConstants.java]: Why unnecessary extra lines ?

Comment (grainier) [inline components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/util/ConfigurationConstants.java]: @mohanvive removed extra lines in #171

**Pull request merged**